project/RootBuild.scala

```diff
@@ -191,6 +191,7 @@ object RootBuild extends Build {
       "org.slf4j" % "slf4j-api" % slf4jVersion,
       "org.slf4j" % "slf4j-log4j12" % slf4jVersion,
       "commons-configuration" % "commons-configuration" % "1.6",
+      "com.google.api-client" % "api-client" % "1.18.0-rc",
```

Comment:
@nhanitvn @Huandao0812 - Hi guys, what's the correct way to add this dependency?(com.google.api-client)

I get cannot resolve dependency error when I try to compile